### ISSUE TITLE
Capabilities test uses a snapshot that's not checked into source control

### ISSUE THREAD

**Issue by richievos** [reef-pi/reef-pi#1849]
[front-end/src/configuration/capabilities.test.js](https://github.com/reef-pi/reef-pi/blob/main/front-end/src/configuration/capabilities.test.js) has:

```
describe('render capabilities component', () => {
  it('<Capabilities />', () => {
    const component = renderer.create(
      <Capabilities capabilities={[]} update={() => {}} />
    )

    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
```

That tests that the capabilities view doesn't change. However, the actual snapshot isn't checked in anywhere, so it's just checking if it's changed since the last time you locally ran the test. It also would be a no-op in source control, because the snapshot doesn't exist so jest will on-the-fly generate it using the results of the test eval, effectively meaning it's a tautology.

Just noting this as a FYI. It's not blocking me on anything, since I just worked around the local failure by `rm -rf ./front-end/src/configuration/__snapshots__` to cause my local to regen it.

If you'd like to keep this test, I am happy to regen it and commit the snapshot.

**Comment by ranjib** [comment]
@richievos agree. I think i was hitting a standard (lint) issue with the snapshots and had to delete the snapshot directory. I concur we should either checkin the snapshot or just delete the test . I'll see if i can fix the standard issue, then i can check in the snapshot, else i'll delete the test. Any help is welcome as usual :-)
thank you for bringing this up.
